bin/codecept.js

```diff
@@ -156,18 +156,39 @@ program.command('run-multiple [suites...]')
 
   .action(require('../lib/command/run-multiple'));
 
-program.command('info')
+program.command('info [path]')
```

Comment:
Can we use prarms same as in another commands?
Example, for config use `-c or --config`